news_items/037_feedback_view/index.html

```diff
@@ -0,0 +1,73 @@
+<style>
+ img {
+     display: block;
+     margin-left: auto;
+     margin-right: auto;
+     margin-top: 15px;
+     margin-bottom: 15px;
+ }
+ pre {
+     width: 90%;
+     margin-left: auto;
+     margin-right: auto;
+     overflow: hidden;
+     margin-bottom: 1rem;
+     font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+     background: #f0f0f0;
+     padding: 1em;
+     padding-bottom: 0em;
+ }
+</style>
+
+<p>
+  Thanks to the work of numerous contributors, the PrairieLearn
+  development team is looking forward to introducing improvements in
+  the instructor's experience for manual grading questions in the
+  coming months. As part of these improvements, some changes in visual
+  elements for the student's question view become necessary. So if you
+  have any questions that use manual grading, you may need to review
+  some of your questions.
+</p>
+<p>
+  One of the features of manual grading is the ability to provide
+  open-ended feedback to students, which allows students to understand
+  their misconceptions and improve their learning experience. This
+  feedback is typically provided
+  using <a href="https://prairielearn.readthedocs.io/en/latest/manualGrading/#uploading-the-scores-and-feedback"
+  target="_blank">the grade upload feature</a>, and so far required
+  question creators to include it explicitly in
+  the <code>question.html</code> file. However, a recent change has
+  updated the student's question page to show this feedback by default
+  instead.
+</p>
+<p>
+  <img src="manualGradingFeedback.png" class="border rounded w-75" alt="The feedback content is shown in a card just above the submission panel.">
+</p>
+<p>
+  One caveat of the change above is that, if
+  the <code>question.html</code> file continues to include the
+  feedback in the submission panel, then feedback will be shown to
+  students twice, once in the new panel, and once where it was shown
+  before. To avoid this duplication, instructors are strongly
+  encouraged to remove any reference to the manual feedback text from
+  their questions.
+</p>
+<p>
+  For most instructors using manual feedback, the question will
+  include a reference to <code>{{{feedback.manual}}}</code> somewhere
+  in the question file. For reference, the manual grading
+  documentation had the following snippet as a suggestion. You are
+  encouraged to remove this snippet from your questions.
+</p>
+<pre class="border rounded bg-light p-2">
+&lt;pl-submission-panel&gt;
+  {{#feedback.manual}}
+  &lt;p&gt;Feedback from course staff:&lt;/p&gt;
+  &lt;markdown&gt;{{{feedback.manual}}}&lt;/markdown&gt;
+  {{/feedback.manual}}
+&lt;/pl-submission-panel&gt;
+</pre>
```

Comment:
This is all fine, but it takes until the very end of the news item to tell instructors what to do. Could we get this up in the first paragraph? Say something roughly like "There will be changes coming to manual grading in the coming months. The first change is that manual grading feedback will be shown automatically to students. If you have been using manual grading feedback then you can remove <code>{{{feedback.manual}}}</code> from your questions to avoid showing it to students twice. More details are below."

Reply:
I have reviewed the first few paragraphs with this in mind. Let me know if it looks clearer.

Reply:
Looks great -- thanks!